handbook/engineering/README.md

```diff
@@ -749,7 +749,9 @@ To target a different version of Fleet, use the `--tag` flag to target any tag i
 
 To start a preview without starting the simulated hosts, use the `--no-hosts` flag (e.g., `fleetctl preview --no-hosts`).
 
-For each bug found, please use the [bug report template](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=bug%2C%3Areproduce&template=bug-report.md&title=) to create a new bug.
+For each bug found, please use the [bug report template](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=bug%2C%3Areproduce&template=bug-report.md&title=) to create a new bug report issue.
+
+For unreleased bugs in an active sprint, no bug report issue is created. Instead, QA moves the associated story (ignoring any technical sub-task issues) back to the "In progress" column in the appropriate project board. Fixing the bug becomes part of the story.
```

Comment:
@sharon-fdm @lukeheath @zayhanlon @zwass @zhumo

Reply:
Is this stating the obvious or we want to add it?
"..._back to the "In progress" column in the appropriate project board_ **with comments about the bug or lack of expected functionality that was found.**"

Reply:
@sharon-fdm Good thinking!  Makes sense to add it to me.  Maybe we clarify who gets assigned too.  Simplest to remember would be the EM, since that's what we're doing for already-released bugs

Reply:
> Maybe we clarify who gets assigned too

Oh great point! 👍